server/src/services/s3.js

```diff
@@ -0,0 +1,54 @@
+import AWS from "aws-sdk";
+
+export const uploadFileToS3 = async (path, file, fileName, contentType) => {
+  try {
+    if (process.env.REACT_APP_ENVIRONMENT === "development") {
+      AWS.config.update({
+        region: "us-west-1",
+        accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+        secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+      });
+    } else if (process.env.REACT_APP_ENVIRONMENT === "production") {
+      AWS.config.update({
+        region: "us-west-1",
+      });
+    }
+
+    const s3 = new AWS.S3();
+
+    const params = {
+      Bucket: path,
+      Key: fileName,
+      Body: file.buffer,
+      ContentType: contentType,
+    };
+
+    await s3.upload(params).promise();
+  } catch (error) {
+    console.log("fail load file to s3");
+    console.log(error.message);
+    return res.status(401).send({ message: error.message });
+  }
+};
+
+export const delFileFromS3 = async (path) => {
+  try {
+    AWS.config.update({
```

Comment:
Don't you need to update configuration based on whether it's development mode or production here?

Reply:
@lnguyen2693 config same as uploadToS3